src/behandlinger/behandlingsresultattyper.js

```diff
@@ -35,6 +35,14 @@ const behandlingsresultattyper = [
   {
     kode: 'MEDLEM_I_FOLKETRYGDEN',
     term: 'Medlem i folketrygden'
+  },
+  {
+    kode: 'AVSLUTTET_UTEN_ENDRING',
```

Comment:
Pga. ny term lurer jeg nå på om det blir mulig å avslutte behandlingen med endringer som ikke er et nytt vedtak.
Ved du det?
Hvis man kan faktisk endre noe uten å produsere nytt vedtak er kanskje `AVSLUTTET_UTEN_ENDRING` en misvisende kode.

Reply:
`AVSLUTTET_UTEN_ENDRING` er kun tilgjengellig for `NY_VURDERING`. Det vil si at vi aldri setter denne resultattypen hvis vi er på førstegangsbehandling.

Reply:
Det gir noen kontekst men svarer ikke helt på mitt spørsmål. Mao kan du "avslutte uten endring" i en ny vurdering og gjøre endringer som ikke er et nytt vedtak?

Reply:
Tror jeg forstår nå litt mer hva du mener. Ja, det stemmer at meningen bak denne resultattypen er at man avslutter selve behandlingen, men endrer _ikke_ på fagsaksstatus til noe nytt. Status forblir som det er, uansett hva forrige status er. Om man kan gjøre _andre_ endringer på behandlingen/fagsaken i mellomtiden, utenom status, før man avslutter ny vurdering med denne resultattypen vites ikke. 

Hva foreslår du?

Reply:
> Om man kan gjøre andre endringer på behandlingen/fagsaken i mellomtiden, utenom status, før man avslutter ny vurdering med denne resultattypen vites ikke.

Det var det jeg lurte på 🙂 
Hvis fag er klar over dette så ville jeg foreslå `AVSLUTTET_UTEN_NYTT_VEDTAK` eller tilsvarende til fag. Hvis ikke så ville jeg ta dette opp.
Poenget er at "UTEN_ENDRING" kan skape misforståelser for teamet, spesielt utviklere, langt frem i tid

Reply:
Har snakket med fag igjen. 

Fag er ikke enige og ønsker fremdeles å bruke AVSLUTTET_UTEN_ENDRING.